src/renderer/ScreenController.tsx

```diff
@@ -89,10 +90,16 @@ export default class ScreenController extends Component {
   }
 
   async selectAccount(accountId: number) {
+    // for now we still need to call the backend function,
+    // because backend still has sleected account
     await DeltaBackend.call('login.selectAccount', accountId)
+
     this.selectedAccountId = accountId
-    const account = await DeltaBackend.call('login.accountInfo', accountId)
-    if (account.type === 'configured') {
+    ;(window.__selectedAccountId as number) = accountId
+    const account = await BackendRemote.rpc.getAccountInfo(
```

Comment:
Not a big thing of course, but I would like if the way to call the backend methods would be simply `Backend.getAccountInfo()` or is there any specific reason for the `.rpc.`? Also I think Backend already means in our architecture that it's somewhat remote.

Reply:
rpc is the raw autogenerated stuff (from `deltachat-jsonrpc/typescript/generated/client.ts`), other functions on `BaseDeltaChat` are implemented manually:
`deltachat-jsonrpc/typescript/src/client.ts#L43`: 


https://github.com/deltachat/deltachat-core-rust/blob/83d71068f7335574e9547bea441bfc5ccd81f9e0/deltachat-jsonrpc/typescript/src/client.ts#L43-L55

Reply:
yeah we could rename `BackendRemote` to `Backend` (just wanted something different than `DeltaRemote`)